Documentation/README_Solver.md

```diff
@@ -154,6 +156,26 @@ To force the associated GameObject to stay vertical in any mode other than *None
 
 ![SurfaceMagnetism Example](../Documentation/Images/Solver/SurfaceMagExample.png)
 
+### HandConstraint and HandConstraintPalmUp
```

Comment:
This heading lists both **HandConstraint** and **HandConstraintPalmUp**. The section does mention **HandConstraintPalmUp** but doesn't document it. I know it references a sample scene, but could it be documented here as well? If **HandConstraintPalmUp** inherits directly from **HandConstraint** and simply adds a few more properties, that delta could simply be added to the bottom of this section.

Reply:
Hi, the paragraph does mention "A derived class of [`HandConstraint`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.HandConstraint) called [`HandConstraintPalmUp`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.HandConstraintPalmUp) is also included to demonstrate a common behavior of activating the solver tracked object when the palm is facing the user. "

Were you looking for something else? If so, please feel free to open a PR against mrtk_development. This PR is meant to be just a cherry pick of an existing PR into our stabilization branch, so would like to keep extra changes to a minimum, if possible, to avoid confusion between branches.